extensions/wikia/Email/templates/contributionAppreciation.mustache

```diff
@@ -0,0 +1,10 @@
+{{> greeting}}
+<table>
+	<p>{{appreciationInfo}}</p>
+	<blockquote>{{appreciationText}}</blockquote>
```

Comment:
Normally this hardcoded order would be a problem (for languages with different ordering). For EN experiment this will suffice.